lib/committee/schema_validator/option.rb

```diff
@@ -36,7 +36,12 @@ def initialize(options, schema, schema_type)
         @check_header        = options.fetch(:check_header, true)
         @coerce_recursive    = options.fetch(:coerce_recursive, true)
         @optimistic_json     = options.fetch(:optimistic_json, false)
-        @parse_response_by_content_type = options.fetch(:parse_response_by_content_type, false)
+        @parse_response_by_content_type = if options[:parse_response_by_content_type].nil?
+          Committee.warn_deprecated('Committee: please set parse_response_by_content_type = false because we\'ll change default value in next major version.') 
+          false
+        else
+          options.fetch(:parse_response_by_content_type)
+        end
```

Comment:
hmm, maybe @ota42y has a better answer, but since we do not practice branching per major versions, and that we have already released version 4.0.0 (major version), i wonder if we can simply have `parse_response_by_content_type` use `false` by default?
> in other words, the new `parse_response_by_content_type` is only available for >= 4.0.0 onwards? 

in any case, i think the message :
> because we'll change default value in next major version.
can be improved by saying what major version that would be.

Reply:
I think we should show error message.

This change is better behavior, beneficial for many new users so we should set `true` by default
But we can't break backward compatibility for now users....

So I think we set `false` by default in this version and show warning message the user doesn't set value.
If the user responds to all warning messages, changing the default value will not affect the user, so no one is in trouble.